src/index.ts

```diff
@@ -9,3 +9,5 @@ export {
   responseInterface
 } from './types'
 export default useSWR
+
+export { cacheClear } from './config'
```

Comment:
I would personally prefer to export this as a function called `clearCache`. But I'm happy either way.